src/slick.grid.ts

```diff
@@ -2326,7 +2327,7 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
 
   protected createCssRules() {
     this._style = document.createElement('style');
-    this._style.nonce = 'random-string';
+    this._style.nonce = this.options.nonce || '';
```

Comment:
can you use it from `this._options` instead (with the underscore `_`) because the `this.options` might not be in sync if for example you call `grid.setOptions({..})`. I think we'll have to maybe remove the `this.options` in the future since it's not very clear what the difference is, but for now can you just please update your PR

Reply:
actually nevermind, I just made the change myself to your PR because I want to push a new version right after